Functions/xASL_stat_GetROIstatistics.m

```diff
@@ -177,7 +177,7 @@
 
 %% Define number of sessions to use
 
-[nSessions, bSessionsMissing] = xASL_adm_GetPopulationSessions(x); % obtain number of Sessions by determining amount of input files present in the Population folder
+[nSessions, bSessionsMissing, x.SESSIONS] = xASL_adm_GetPopulationSessions(x); % obtain number of Sessions by determining amount of input files present in the Population folder

```

Comment:
Is this third new output parameter `x.SESSIONS` used? I'm not sure if we should avoid this, since `x.SESSIONS` is also used in the Structural and ASL modules.